src/main/kotlin/no/nav/syfo/aksessering/api/InternalSykmeldingApi.kt

```diff
@@ -0,0 +1,28 @@
+package no.nav.syfo.aksessering.api
+
+import io.ktor.application.call
+import io.ktor.http.HttpStatusCode
+import io.ktor.response.respond
+import io.ktor.routing.Route
+import io.ktor.routing.get
+import io.ktor.routing.route
+import no.nav.syfo.aksessering.SykmeldingService
+import no.nav.syfo.aksessering.tilgang.TilgangskontrollService
+
+fun Route.registrerInternalSykmeldingApi(sykmeldingService: SykmeldingService, tilgangskontrollService: TilgangskontrollService) {
+    route("/api/v1/internal") {
+        get("/sykmeldinger") {
+            val token = call.request.headers["Authentication"]?.removePrefix("Bearer ")
+            if (token == null) {
+                call.respond(HttpStatusCode.Unauthorized)
+            } else {
+                val fnr = call.request.queryParameters["fnr"]
+                when {
+                    fnr.isNullOrEmpty() -> call.respond(HttpStatusCode.BadRequest, "Missing query param: fnr")
+                    !tilgangskontrollService.hasAccessToUser(fnr, token) -> call.respond(HttpStatusCode.OK, emptyList<InternalSykmeldingDTO>())
```

Comment:
Er dette riktig oppførsel hvis veileder har slått opp på en bruker de ikke har tilgang til? Bør vi ikke gi en 403 eller noe tilbake i stedet for en ok og en tom liste? Jeg gjetter på at Modia vil vise dette som at brukeren ikke har noen sykmeldinger :)

Jeg synes kanskje også at vi bør snu om på logikken sånn at vi sier at hvis man har tilgang til bruker -> returner sykmeldingene, else -> returner 403 eller tom liste. Grunnen til at jeg synes det er både fordi det ofte vil være lettere å lese (en ! foran uttrykket kan være litt vanskelig å se), men også fordi det føles tryggere at default-oppførsel (else'en) er at du ikke får tilgang.

Reply:
Ja den kan jeg snu :) 
Ja usikkert på om det skulle være 403 Forbidden eller 200 Ok med tom liste. Skal høre med veden om det